app/modules/scan/service.server.ts

```diff
@@ -34,7 +37,7 @@ export async function createScan(params: {
     };
 
     /** If user id is passed, connect to that user */
-    if (userId) {
+    if (userId && userId != "anonymous") {
```

Comment:
I am not sure what the "anonymous" is. I dont see us using this anywhere or passing it to `createScan`.

Reply:
NVM, i found it in some old code. Didnt even know we handle it like this :)